container-storage-setup.1

```diff
@@ -41,6 +64,22 @@ their default settings. Anything user wants to change, should be changed
 in the INPUTFILE. This is the file which will
 override any settings specified in /usr/share/container-storage-setup/container-storage-setup.
 
```

Comment:
Suggested nit fix.  Change 'configuration in INPUTFILE or /etc/...' to 'configuration in the INPUTFILE or in /etc/...'.  Also "Anything user wants to change,' to 'Anything that the user wants to change'